go/vt/vttablet/onlineddl/schema.go

```diff
@@ -558,4 +565,8 @@ var ApplyDDL = []string{
 	alterSchemaMigrationsTableLogFile,
 	alterSchemaMigrationsTableRetainArtifacts,
 	alterSchemaMigrationsTablePostponeCompletion,
+	alterSchemaMigrationsTableRemovedUniqueNames,
+	alterSchemaMigrationsTableRemovedNoDefaultColNames,
+	alterSchemaMigrationsTableExpandedColNames,
+	alterSchemaMigrationsTableRevertibleNotes,
```

Comment:
These new column in `schema_migrations` table help us to understand better the nature of the schema change, and in particular inform us what may be blockers/issues to reverting the migration.